content/services/support.rst

```diff
@@ -1,12 +0,0 @@
-:nosearch:
-
-=======
-Support
-=======
-
-.. toctree::
-   :titlesonly:
-
-   support/where_can_i_get_support
-   support/what_can_i_expect
-   support/supported_versions
```

Comment:
@fpodoo 

   support/what_can_i_expect
   support/supported_versions

We need to keep at least these pages (daily used in both tech and functional support to communicate with customers), it cannot be removed from the doc like this (it is used in in page odoo.com/help but it is not enough, it needs to be accessible from the main website so customers can find this information by themselves)

cc @ALE-Odoo

Reply:
@Feyensv @alt-odoo @ALE-Odoo 

@fpodoo 's answer to my similar comment last week was:
- "what can I expect" is actually in the Odoo contract which is online. If you share a link, it should be to that page specifically. According to him, what's in our doc wasn't exactly the same as what's in the contracts. Else, the info should be visible on odoo.com/help and not in the documentation
- "supported versions" isn't deleted but moved to the "Administration" section.

Reply:
Yes, the documentation is not the right place for that and it's risky to change the wording from the contract as it's very subjective. I don't agree neither on what was in the content of this page, it was weird to have a Chapter about "Services" in the product documentation.

Reply:
@fpodoo 

I agree to centralize this in the contract part, but saying : 
"it's risky to change the wording from the contract as it's very subjective" is non-sense.

Words in a contract should be crystal clear for everyone to avoid misunderstandings, which was most probably the initial goal of this doc page.

We should take this opportunity to change this and make it better for everyone.

Reply:
@fpodoo 

Some customers (one app free, trials) don't have a contract moreover, the previous page was useful for them too to manage their expectations. What do we do with them ?